configuration/pih/liquibase/liquibase-mexico.xml

```diff
@@ -19,5 +24,6 @@
                 and v.voided = 0;
         </sql>
     </changeSet>
+-->
```

Comment:
I don't think this ever worked?  I'm commenting this out because I don't what it inadvertently running at this point without further testing.